hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -168,6 +169,11 @@ enum FreonOps {
       defaultValue = "1")
   private int numOfValidateThreads = 1;
 
+  @Option(names = {"--validate-channel"},
+      description = "grpc or short-circuit.",
+      defaultValue = "grpc")
+  private String validateChannel = "grpc";
```

Comment:
- I think `--validation-channel` would be a better name, since we are not validating the channel.
- Make it an `enum` with possible values for the option.

Reply:
Option name changed. For the enum suggestion, I would prefer to keep using the String. Thanks.

Reply:
> For the enum suggestion, I would prefer to keep using the String.

Can you please explain the reason?

Reply:
There is no existing enum which includes both grpc and short-circuit, and I also has no intend to create a new enum for that.

Reply:
https://picocli.info/#_enum_types